schedule/yast/yast2_gui/yast2_gui_sle-svirt-hvm.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name:           yast2_gui
+description:    >
+    Test for yast2 UI, GUI only.
+    Running on created gnome images which provides both text console for ncurses UI tests as well
+    as the gnome environment for the GUI tests.
+vars:
+    BOOTFROM: c
+    HDDSIZEGB: 20
+    SOFTFAIL_BSC1063638: 1
+    VALIDATE_ETC_HOSTS: 1
```

Comment:
after merge it, let's clean the settings not needed in [openQA test suites](https://openqa.suse.de/admin/test_suites)